pyproject.toml

```diff
@@ -122,6 +122,7 @@ coverage = ">=6.2"
 codacy-coverage = ">=1.3.11"
 coveralls = ">=3.3.1"
 docutils = ">=0.16,<1"
+fakeredis = "^2.21.3"
```

Comment:
I think `snovault` already provides a primitive for this? or perhaps it is in utils... either way I know there is another mechanism built into pytest that I think can work

Reply:
It complained without this.